tests/weight_converter/torch/test_torchscript.py

```diff
@@ -3,20 +3,10 @@
 
 
 @pytest.mark.skipif(pytest.skip_torch, reason="requires pytorch")
```

Comment:
Do we still need skips?

Reply:
actually wherever specific test model groups are used the skip is still neccecarry as those groups are not shrinking to length one and will fail to provide the model (as it is not loaded).

maybe we can make those then empty lists instead, if the tests then pass/skip (haven't tried that yet)

Reply:
Ok, I don't understand the new logic in the tests yet, could you quickly summarise it, or even better write a comment in conftest.py to explain it?

Reply:
I'll add a comment...

Reply:
just tested the empty list in a parametrized fixture and it works, meaning it just skips the test (and marks it skipped as well), so I'll use that instead of the skip flags where possible